core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -88,12 +93,16 @@ ${h.initPageVariables(context)}
     </j:if>
 
     <title>${h.appendIfNotNull(title, ' [Jenkins]', 'Jenkins')}</title>
+    <link rel="stylesheet" href="${resURL}/css/layout-common.css" type="text/css" />
     <link rel="stylesheet" href="${resURL}/css/style.css" type="text/css" />
     <link rel="stylesheet" href="${resURL}/css/color.css" type="text/css" />
     <link rel="stylesheet" href="${resURL}/css/responsive-grid.css" type="text/css" />
     <j:if test="${attrs.css!=null}">
       <link rel="stylesheet" href="${resURL}${attrs.css}" type="text/css" />
     </j:if>
+    <j:if test="${empty(attrs.sidepanel) or attrs.sidepanel}">
```

Comment:
Yes, I did it that way so that backward compatibility is guaranteed. In fact, `left-side` + `main-content` (two-column) is the default layout.